packages/internal-test-helpers/lib/engine.js

```diff
@@ -0,0 +1,9 @@
+import Engine from '@ember/engine';
```

Comment:
Ultimately, I think we shouldn't need this file. Just extending `Engine` shouldn't necessarily trigger a deprecation...

Reply:
Yes, it does, if you don't override the Resolver property to something other than the DefaultResolver.